### DISCUSSION TITLE
Hope to see the project integrated with interfaces such as AnythingLLM.

### DISCUSSION THREAD

**Discussion by ss958** [open-webui/open-webui#3157] [General]
Hope to see the project integrated with interfaces such as AnythingLLM.

**Comment by sxueck** [comment]
It's strange. Anythingllm provides an API interface named OpenAI, but when I put it to "external connection", this causes an error because Anythingllm provides a non-standard JSON return. I now need to write a middle layer converter.

**Comment by sxueck** [comment]
anythingllm responce:

```
{
  "models": [
    {
      "name": "Sample workspace",
      "model": "sample-workspace",
      "llm": {
        "provider": "ollama",
        "model": "llama3:8b"
      }
    },
    {
      "name": "Second workspace",
      "model": "workspace-2",
      "llm": {
        "provider": "openai",
        "model": "gpt-3.5-turbo"
      }
    }
  ]
}
```

any cause line:
https://github.com/open-webui/open-webui/blob/99d10d1189452ad49fcace219e9c90ae65906cd1/backend/apps/openai/main.py#L216

**Comment by ss958** [comment]
I hope the official can add compatibility with several mainstream non open API interfaces. Alternatively, open webui can develop its own workflow function engine similar to Dify or MAXKB.
